src/components/alert/bl-alert.css

```diff
@@ -0,0 +1,103 @@
+:host {
+  --bl-display: block;
+
+  display: var(--bl-display);
+}
+
+.alert {
+  --bl-padding: var(--bl-size-m);
+  --bl-main-color: var(--bl-color-primary);
+  --bl-main-bg-color: var(--bl-color-accent-primary-background);
+  --bl-description-font-size: var(--bl-font-size-m);
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--bl-main-bg-color);
+  color: var(--bl-color-content-primary);
+  box-shadow: 0 0 0 1px var(--bl-main-color);
+  border-radius: var(--bl-border-radius-s);
+  padding: 0 var(--bl-padding);
+}
+
+.description {
+  font-size: var(--bl-description-font-size);
```

Comment:
I think description has it's own `font` style. You should not need to set size and line-height manually.